elements/pfe-jump-links/demo/demo.css

```diff
@@ -7,7 +7,12 @@ html {
     --pfe-navigation--Height--actual: 55px;
     --pfe-jump-links--nav-height: 62px;
     --pfe-jump-links-panel__section--spacer: 117px;
-    --pfe-jump-links-panel--offset: 120px;
+    --pfe-jump-links-panel--offset: 55px;
+
+    /* For testing */
```

Comment:
These should probably be sass comments or removed

Reply:
It's a demo css file - they're there to toggle on or off for testing in the demo page.